### ISSUE TITLE
add more maintainers

### ISSUE THREAD

**Issue by freddii** [jnk22/kodinerds-iptv#406]
At the moment pull requests do not get merged fast.
It seams only  person that has the right to merge.
Maybe it would help to give some other trustful persons the rights to merge pull requests.

**Comment by jnk22** [comment]
I would love to do that!

At this moment, users with write access are @SkyBird1980, @tzwn, and me.
Both of them helped and supported me at the beginning when we have set this all up.

I have invited you as a contributor now, too.

**If there is anyone else that has provided at least some valid changes in the past, feel free to use this thread/issue to ask for contribution permissions.**

---

@freddii If you accept the invitation and think that we should discuss some guidelines on what content should get on these lists, feel free to do that here.

To state a few 'main ideas' that I have had since this was all set up:
- only legal streams (as this is in some way coupled to [kodinerds.net](https://www.kodinerds.net/index.php/Thread/56713) - neither them, nor me, want any connection to illegal stuff - this has worked great so far with everyone involved here at GitHub!)
- mainly represent all TV/radio streams that are also present at 'normal TV' receivers and are at least coupled to a 'main' TV/radio station (so at least all DVB-S, DVB-C, DVB-T2 are always valid streams for this list and special streams if their main station exists in the former group)
- direct high quality streams > master streams* (eg. `index_3096_av-p.m3u8` vs `master.m3u8`)

*The reason for this have been the faster channel switches in the past. This however requires frequent changes to stream URLs as most of you have seen in the past. The 'master' link should be working even when a direct stream URL would have to be changed.

Unfortunately I do not really see a possibility on GitHub to provide 'both'.

As I am not really using these lists myself anymore (at least for the time being), I am open to further discuss or change this approach.
A few months ago there has been a small discussion about it at the Kodinerds forum, where I felt like it was somehow 50/50 regarding this.

---

To sum it up, I am more than happy to have more people involved in this with direct write access, especially since I am not an active user of these lists myself anymore.

**Comment by freddii** [comment]
i think, as long as there is nobody maintaining the scripts regularly, it is a good idea to use `master.m3u8` instead of `index_3096_av-p.m3u8`

**Comment by Megachip** [comment]
I'll guess 4 maintainers should enough... at least for getting some feedback ;)

**Comment by SkyBird1980** [comment]
if you want to exchange me i'll have no problem with it. i haven't merged any PR since long time.

**Comment by bjoerns1983** [comment]
I think @tzwn could be removed. Last activity at Github was in 2017 and the last time he was at the Forums was was mid 2020.

**Comment by issteve** [comment]
I suggest for now: @Mynacol if they like to do so...
